pom.xml

```diff
@@ -830,10 +826,15 @@
         <carbon.identity.framework.imp.pkg.version.range>[5.17.5, 6.0.0)
         </carbon.identity.framework.imp.pkg.version.range>
 
-        <carbon.identity.organization.management.version>1.0.54</carbon.identity.organization.management.version>
+        <carbon.identity.organization.management.version>1.1.5</carbon.identity.organization.management.version>
         <carbon.identity.organization.management.version.range>[1.0.46, 2.0.0)
```

Comment:
Doesn't lower bound have to be changed too?

Reply:
updated